CGF.py

```diff
@@ -0,0 +1,13 @@
+
+#given a list find the greatest commmon factor,( the largest number that is divisible by all 
+# items in the list
+
+def cdf(arr):
+	max_num = max(arr)
+	for i in range(max_num +1,1,-1):
+		if all(j%i == 0 for j in arr):
+			return i
+
+print(cdf([9,3,15]))
```

Comment:
When you include test cases, you need to provide not only the input, but the expected result as well.
Normally you would do something like
```assert cdf([9, 3, 15]) == <expected value>```